core/util.py

```diff
@@ -58,13 +59,36 @@ def group_by_first_element(tuples: list[tuple]) -> dict[str, list[tuple | Any]]:
     return nested
 
 
-def get_project_number(project_id):
-    """Extracts the project number from a project ID.
+def get_project_number_by_cell(row_index: int, sheet: str):
```

Comment:
I think `get_project_number_by_position` is more accurate? Cell to me is both row and column indexes.